README.md

```diff
@@ -1,19 +1,38 @@
-# Kinde Elixir SDK
+# Kinde Elixir
 
 The Kinde SDK for Elixir.
 
-You can also use the Elixir starter kit [here](https://github.com/kinde-starter-kits/elixir-starter-kit).
+You can also use the Elixir starter kit here https://github.com/kinde-starter-kits/elixir-starter-kit.
```

Comment:
@aliaamuqaddas see an example here the words should be hyperlinked https://github.com/kinde-oss/kinde-php-sdk#kinde-php-sdk